src/vaishnavi/Facebook.java

```diff
@@ -0,0 +1,84 @@
+package vaishnavi;
+
+import org.openqa.selenium.By;
+
+/* Selenium Assignment 2 : 17th Aug 2020
+
+1. Navigate to facebook sign in page using url https://www.facebook.com/
+2. Verify page title is �Facebook � log in or sign up�.
+3. Enter email id & password, and click on �Log In� button.
+4. Verify page title is �Facebook�.
+5. Now using navigation command go to https://www.google.com/
+6. Verify when user navigate back at that time page title is �Facebook�.
+7. Verify when user navigate forward at that time page title is �Google�
+8. Verify when user refresh the page, title remains same.  */
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Facebook {
+
+	void verifyFacebookPage() throws InterruptedException {
+
+		WebDriver driver = new ChromeDriver();
+		driver.navigate().to("https://www.facebook.com/");
+		driver.manage().window().maximize(); // Maximizing the window
+
+		if (driver.getTitle().equals("Facebook � log in or sign up"))
+			System.out.println("You are on a correct page");
+		else
+			System.out.println("You are on a wrong page with name " + driver.getCurrentUrl());
+
+		driver.findElement(By.id("email")).sendKeys("9545336556");
+		driver.findElement(By.id("pass")).sendKeys("vaish@78#");
+		driver.findElement(By.name("login")).click();
+
+		// Verify Page title
+		Thread.sleep(3000);
+		if (driver.getTitle().equals("(2) Facebook"))
```

Comment:
Do you know why title is coming as "(2) Facebook" , bez you have 2 unread notification. what if there will multiple in future.. You should use contains method.

Reply:
- Thank you for the suggestion and this information. Implemented it in the code.